isamples_metadata/SESARTransformer.py

```diff
@@ -301,17 +305,39 @@ def has_context_categories(self) -> typing.List[str]:
             material_type, primary_location_type
         )
 
-    def has_material_categories(self) -> typing.List[str]:
-        material = self._material_type()
-        if not material:
+    def _compute_material_prediction_results(self) -> typing.Optional[typing.List[PredictionResult]]:
```

Comment:
Need to refactor this into something that will work generally across multiple category/confidence pairs.  Will do that work once we have the next model combo to integrate.